wlm_server/team/models.py

```diff
@@ -0,0 +1,4 @@
+from django.db import models
+
+class Team(models.Model):
+    name = models.CharField(max_length=30)
```

Comment:
How about making this unique, just like the `AbstractUser`'s `username`?

Reply:
Oh, I forgot it completely😅 Thanks for notifying me.